package.json

```diff
@@ -81,10 +80,14 @@
     ]
   },
   "jest": {
+    "setupFiles": [
+      "dotenv/config"
+    ],
     "setupFilesAfterEnv": [
       "<rootDir>/jest.setup.js"
     ],
     "preset": "jest-puppeteer",
+    "testEnvironment": "node",
```

Comment:
What's the reason for making node the default instead of JSDOM?

Reply:
It looks like a few browser-based queries fails when the testEnvironment is not set to jsdom. Things like `navigator.mediaDevices` appears undefined when running the unit tests. I'm not entirely sure why this is the case.

By default, the testEnvironment is set to node so this just makes the current test behavior explicit. It does make sense for jsdom to be the default although..

Reply:
The reason why I would lean towards making JSDOM the default environment for this repository is because the code for the adapter is meant to run on browsers